build_artifacts/v1/v1.9/v1.9.0/gpu.env.in

```diff
@@ -0,0 +1,54 @@
+# This file is auto-generated.
+conda-forge::langchain-aws[version='>=0.1.6,<1.0.0']
+conda-forge::jupyter-collaboration[version='>=1.1.0,<2.0.0']
+conda-forge::sagemaker-code-editor[version='>=1.1.0,<2.0.0']
+conda-forge::amazon-sagemaker-jupyter-ai-q-developer[version='>=1.0.3,<2.0']
+conda-forge::amazon-q-developer-jupyterlab-ext[version='>=3.1.0,<4.0']
+conda-forge::amazon_sagemaker_sql_editor[version='>=0.1.7,<1.0.0']
+conda-forge::amazon-sagemaker-sql-magic[version='>=0.1.1,<1.0.0']
+conda-forge::langchain[version='>=0.1.9,<1.0.0']
+conda-forge::fastapi[version='>=0.110.3,<1.0.0']
+conda-forge::uvicorn[version='>=0.30.1,<1.0.0']
+conda-forge::pytorch-gpu[version='>=2.0.0,<3.0.0']
+conda-forge::tensorflow-gpu[version='>=2.14.0,<3.0.0']
```

Comment:
Why did we make this change ?
Is tensorflow-gpu officially maintained ?

Reply:
Yes it's built from the same feedstock as tensorflow (tensorflow-feedstock) and holds gpu-specific builds